docs/guides/integrating-cert-manager.md

```diff
@@ -0,0 +1,290 @@
+
+### Step 2 – Create a ClusterIssuer
+
+Next we need to create a ClusterIssuer. Issuers, and ClusterIssuers, are Kubernetes resources that represent certificate
+authorities (CAs) that are able to generate signed certificates by honouring certificate signing requests. The only
+difference between Issuers and ClusterIssuers is that Issuers are namespaced resources, and ClusterIssuers are not.
+
+We are using the ACME Issuer type, and Let's Encrypt as the CA server. In order for Let's Encypt to verify that we own
+the domain a certificate is being requested for, we must complete "challenges". This is to ensure clients are
+unable to request certificates for domains they do not own. We will configure the Issuer to use a HTTP01 challenge, and
+our Gateway resource that we will create in the next step as the solver. To read more about HTTP01 challenges, see
+[here](https://cert-manager.io/docs/configuration/acme/http01/). Use the following YAML definition to create the
+resource, but please note the `email` field should be updated to your own email address.
+
+```yaml
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-prod
+spec:
+  acme:
+    # You must replace this email address with your own.
+    # Let's Encrypt will use this to contact you about expiring
+    # certificates, and issues related to your account.
+    email: my-name@example.com
+    server: https://acme-v02.api.letsencrypt.org/directory
+    privateKeySecretRef:
+      # Secret resource that will be used to store the account's private key.
+      name: issuer-account-key
+    # Add a single challenge solver, HTTP01 using NKG
+    solvers:
+    - http01:
+        gatewayHTTPRoute:
+          parentRefs: # This is the name of the Gateway that will be created in the next step
+            - name: gateway
+              namespace: default
+              kind: Gateway
+```
+
+### Step 3 – Deploy our Gateway with the cert-manager annotation
+
+Next we need to deploy our Gateway. Use can use the below YAML manifest, updating the `spec.linsteners[0].hostname`
+field to the required value for your environment.
+
+```yaml
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: Gateway
+metadata:
+  name: gateway
+  annotations: # This is the name of the ClusterIssuer created in the previous step
+    cert-manager.io/cluster-issuer: letsencrypt-prod
+spec:
+  gatewayClassName: nginx
+  listeners:
+  - name: http
+    port: 80
+    protocol: HTTP
+  - name: https
+    # Important: The hostname needs to be set to your domain
+    hostname: "cafe.example.com"
+    port: 443
+    protocol: HTTPS
+    tls:
+      mode: Terminate
+      certificateRefs:
+      - kind: Secret
+        name: cafe-secret
+```
+
+It's worth noting a couple of key details in this manifest:
+
+- The cert-manager annotation is present in the metadata – this enables the cert-manager integration, and tells
+- cert-manager which ClusterIssuer configuration it should use for the certificates.
```

Comment:
looks like this line should be part of the list element before, not a new element